ibis/pandas/core.py

```diff
@@ -87,6 +89,9 @@ def execute_with_scope(expr, scope, **kwargs):
     if op in scope:
         return scope[op]
 
+    if context is None:
+        context = ctx.Summarize()
```

Comment:
We might try to make a more clear name for "context". This is more of an aggregation style than anything